target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -191,7 +191,7 @@ get_status_led() {
 		status_led="$board:green:wps"
 		;;
 	rb750gr3)
-		status_led="$board:green:usr"
+		status_led="$board:blue:pwr"
 		;;
```

Comment:
Hi @blogic, the router explicitly labels it as "PWR" on the sticker, would it be preferred to use the label or call it "power"?